Nayara-Oliveira/exercicio-casa/sistema_de_alunas.py

```diff
@@ -0,0 +1,143 @@
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    print("Insira os seguintes dados para cadastrar uma nova aluna:")
+    nome = input("Nome da aluna: ")
+    sobrenome = input("Sobrenome da aluna: ")
+    turma = input("Turma da aluna: ")
+    quantidade_notas = input("Quantidade de notas: ")
+    try:
+        notas = []
+        for i in range(int(quantidade_notas)):
+            try:
+                nota = float(input(f"Insira a nota #{i+1}: "))
+                if nota >= 0 and nota <= 10:
+                    notas.append(nota)
+                else:
+                    print("Entrada inválida. A nota deve ser um número entre 0 e 10.")
+            except ValueError:
+                print("Entrada inválida. Por favor, digite um número.\n")
+        if len(notas) != int(quantidade_notas):
+            notas = []
+    except ValueError:
+        print("Entrada inválida. Por favor, digite um número inteiro.\n")
+    quantidade_aulas = input("Quantidade de aulas: ")
+    try:
+        presencas = []
+        for i in range(int(quantidade_aulas)):
+            presenca = input(f"A aluna esteve presente na aula #{i+1}? Digite P para presente ou A para ausente: ")
+            if presenca == "P":
+                presencas.append(True)
+            elif presenca == "A":
+                presencas.append(False)
+            else:
+                print("Opção inválida. Por favor, escolha uma opção válida (P ou A).")
+        if len(presencas) != int(quantidade_aulas):
+            presencas = []
+    except ValueError:
+        print("Entrada inválida. Por favor, digite um número inteiro\n")
+    nota_participacao = False
+    try:
+        nota_participacao_float = float(input("Nota de participação da aluna: "))
+        if nota_participacao_float >= 0 and nota_participacao_float <= 10:
+            nota_participacao = nota_participacao_float
+        else:
+            print("Entrada inválida. A nota deve ser um número entre 0 e 10.")
+    except:
+        print("Entrada inválida. Por favor, digite um número")
+    
+    if notas and presencas and nota_participacao:
+        dataset[(nome, sobrenome)] = {
+            "Turma": f"Turma {turma}",
+            "Notas": notas,
+            "Presença": presencas,
+            "Participação": nota_participacao
+        }
+        print(f"A aluna {nome} {sobrenome} foi adicionada com sucesso.")
+    else:
+        print("Não foi possível incluir a aluna na base de dados. Tente novamente")
+    
+def consultar_lista_alunas():
+    nomes_completos = dataset.keys()
+    if not nomes_completos:
+        print("Não há registros de alunas cadastradas")
+    else:
+        lista_nomes = [nome_completo[0] for nome_completo in nomes_completos]
+        print("A lista com os nomes das alunas cadastradas é: ")
+        nomes = [print("-", nome) for nome in lista_nomes]
```

Comment:
Estamos atribuindo a impressão a uma variável de forma desnecessária. Podemos simplesmente imprimir em um for. Deste modo, melhoramos a legibilidade e clareza da impressão do nome das alunas e caso queiramos colocar outros tratamentos fica mais fácil.

**Sugestão:**
```suggestion
        for nome in lista_nomes:
            print(f"- {nome}")
```